src/styles/reset.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
```

Comment:
do not apply any rules to the `*` selector except `box-sizing`

Reply:
got it, thanks!